### ISSUE TITLE
Will SportsTracker import and display performance metrics?

### ISSUE THREAD

**Issue by QA-Manager** [ssaring/sportstracker#232]
Some watches calculate additional metrics, particularly if they are paired with something like the Stryd power meter or the Garmin HR-Tri heart rate monitor. These metrics include things such as VO2 Max, Critical Power, lactate threshold, vertical oscillation, etc. Will SportsTracker import and display these?

**Comment by ssaring** [comment]
There are currently no plans for adding such metrics. The examples you named are very specific. Although many device manufacturers provide similar metrics, they are often different in detail. That's why it would be quite hard to parse, interprete and display all of them properly. Some of them are part of the recorded exercises, some needs to be calculated after parsing. So a lot of effort is needed for that.
And the UI of SporsTracker would become quite bloated when adding all variants of all manufacturers.

A primary goal of SportsTracker is simple usability, sometimes it's good to go with the lowest common denominator for that.
